arm/platform-implementations/home.hpp

```diff
@@ -0,0 +1,122 @@
+#include <libhal-rmd/mc_x.hpp>
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+#include <libhal/accelerometer.hpp>
+#include <math.h>
+#define M_PI 3.14156265
```

Comment:
Bruh... We have PI at home. See https://en.cppreference.com/w/cpp/header/numbers

```suggestion
```